runtime/tests/interpreter/import_test.go

```diff
@@ -115,3 +118,150 @@ func TestInterpretVirtualImport(t *testing.T) {
 		value,
 	)
 }
+
+// TestInterpretImportMultipleProgramsFromLocation demonstrates how two declarations (`a` and `b`)
+// can be imported from the same location (address location `0x1`).
+// The single location (address location `0x1`) is resolved to two locations
+// (address contract locations `0x1.a` and `0x1.b`).
+// Each requested declaration is so imported from a a separate program.
+//
+func TestInterpretImportMultipleProgramsFromLocation(t *testing.T) {
+
+	t.Parallel()
+
+	addressLocation := ast.AddressLocation{0x1}
+
+	importedCheckerA, err := checker.ParseAndCheckWithOptions(t,
+		`
+          pub fun a(): Int {
+              return 1
+          }
+
+          pub fun b(): Int {
```

Comment:
In this scenario, there'd be no way to access this `b()` func?

Reply:
Good question!

Correct, this function is only declared to ensure that it is **not** imported in the `import a, b from 0x1` statement.

Likewise, [the function `a` below](https://github.com/onflow/cadence/pull/333/files#diff-f759a5f9b49ae7e692b50eb5926c786cR159) should not be imported either. 

I'll add comments to make this test clearer

Reply:
Improved this in [`36421c5` (#333)](https://github.com/onflow/cadence/pull/333/commits/36421c5375832e8f4de5d26fdf78726b7c530d01)